heapster.go

```diff
@@ -40,7 +40,7 @@ var (
 	argSinkFrequency   = flag.Duration("sink_frequency", 10*time.Second, "Frequency at which data will be pushed to sinks")
 	argCacheDuration   = flag.Duration("cache_duration", 5*time.Minute, "The total duration of the historical data that will be cached by heapster.")
 	argUseModel        = flag.Bool("use_model", false, "When true, the internal model representation will be used")
-	argModelResolution = flag.Duration("model_resolution", 2*time.Minute, "The resolution of the timeseries stored in the model. Applies only if use_model is true")
+	argModelResolution = flag.Duration("model_resolution", 1*time.Minute, "The resolution of the timeseries stored in the model. Applies only if use_model is true")
```

Comment:
A delta is required due to the existing implementation of the StatStore, where values of the past 1 minute are being averaged together before being placed into buckets. The Get and Last methods of the StatStore expose that last one minute and we are, therefore, interested in its accuracy.

By having the 25 second delta, the values of the last one minute represent a larger set of samples than when not having an offset.

For example, if Updates were happening at every resolution, then the very last resolution of every metric might represent only one actual sample that was used to generate it, depending on the alignment of metrics flowing in.

If no delta is used, then a mechanism would have to be added to force each metric to lag behind in time for the appropriate duration.
For example, a container would be lagging by 1 resolution, its parent pod would be lagging by 2 resolutions and its parent namespace would be lagging behind by 3 resolutions.